src/coreclr/jit/hwintrinsic.cpp

```diff
@@ -1731,6 +1731,8 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
                 {
                     case NI_Sve_ConvertToInt32:
                     case NI_Sve_ConvertToUInt32:
+                    case NI_Sve_ConvertToInt64:
```

Comment:
I expected to have entries for `ConvertToSingle` and `ConvertToDouble` here as well. Did you confirm this works for scenario where `maskSize != operSize` in lowering?